webcompat/form.py

```diff
@@ -295,7 +295,9 @@ def get_problem_summary(category):
     else:
         # Return the usual message in lowercase
         # because it is not at the start of the summary.
-        return get_radio_button_label(category, problem_choices).lower()
+        problem_type_stripped = re.sub(clean, '', get_radio_button_label(
```

Comment:
see my previous comment about using clean. What the code is trying to do here?
Why not working directly on `get_radio_button_label` function  if the issue is there in the first place?